lib/html5_f.ml

```diff
@@ -32,11 +32,16 @@
 
 open Html5_types
 
-module Make
+module Make'
```

Comment:
Please use something more descriptive than `Make'`. Same for `Conv`.

Maybe `Constants` and `Make_with_constants` ?